src/client/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -235,6 +236,9 @@ private Uri CreateAuthorizationUri(bool addPkceAndState = false)
                 authorizationRequestParameters[OAuth2Parameter.Prompt] = _interactiveParameters.Prompt.PromptValue;
             }
 
+            authorizationRequestParameters[TelemetryConstants.XClientLastRequest] = ServiceBundle.TelemetryManager.FetchAndResetPreviousHttpTelemetryContent();
```

Comment:
I though we are sending this as headers? Here you are sending this as form params. Also, is it confirmed that the server accepts telemetry on the /authorize endpoint?

Reply:
as we discussed...only sending to /token endpoint now

---
In reply to: [325635909](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1389#discussion_r325635909) [](ancestors = 325635909)

Reply:
![image](https://user-images.githubusercontent.com/19942418/65208671-ad36de00-da4a-11e9-8093-d33dc7edd652.png)